packages/react-native-worklets/Common/cpp/worklets/SharedItems/Shareables.cpp

```diff
@@ -173,6 +173,20 @@ jsi::Value makeShareableHostObject(
   return ShareableJSRef::newHostObject(rt, shareable);
 }
 
+jsi::Value makeShareableTurboModuleLike(
+    jsi::Runtime &rt,
+    const jsi::Object &object) {
+  auto getPrototypeOf = rt.global()
+                            .getProperty(rt, "Object")
+                            .asObject(rt)
+                            .getPropertyAsFunction(rt, "getPrototypeOf");
+  auto objectPrototype = getPrototypeOf.call(rt, object).asObject(rt);
```

Comment:
Maybe let's avoid assigning intermediate values like `getPrototypeOf` and just call it instead?
```suggestion
                            .getPropertyAsFunction(rt, "getPrototypeOf")
                            .call(rt, object)
                            .asObject(rt);
```